Create tizen-9-0-m1.md (#1977)
Samsung/tizen-docs: docs/platform/release-notes/tizen-9-0-m1.md

# Tizen 9.0 M1 Release

Release date: May 31, 2024

## Release details

- [Getting source code](http://review.tizen.org/git/) (Tizen 9.0 M1 source codes are under **tizen** branch.)

- Getting binaries and images
  - Base: https://download.tizen.org/releases/milestone/TIZEN/Tizen/Tizen-Base/tizen-base_20240516.070242/
  - Profile(Unified): https://download.tizen.org/releases/milestone/TIZEN/Tizen/Tizen-Unified/tizen-unified_20240523.040324/

- [How to flash to a device](../developing/flashing.md)

## Release notes

### System (kernel and system framework)

#### New and changed features

- Kernel
  - Kernel for Raspberry Pi 4 has been upgraded to version 6.6.17.
  - Raspberry Pi 5 has been supported with the ARM64 Raspberry Pi 4 BOOT image
    - Known issue: Raspberry Pi 5 board should have EEPROM version 2013-12-06.

- System and resource management
  - The ISU of resourced package has been added.

- Device and sensor management
  - The priority-based sound feedback playback has been added and the getter of supported theme ID list.

>   - The key mapping between linux kernel input key code and button has been supported when adding new hardware button.

Reviewer: ```suggestion
  - The key mapping between Linux kernel input key code and button has been supported when adding new hardware button.
```

  - The getter of power lock count and callback for change of power lock count have been added.

- OS upgrade
  - The RPK format of ISU package has been added. It is possible to install/uninstall ISU packages via pkgcmd tool.
  - The update-control plugin has been supported in order to separate the vendor/product dependent code.

> - kmod
>   - Version 30 has been upgaded to version 31.
> - f2fs-tools
>   - Version 1.14.0 has been upgaded to version 1.16.0.
> - libdrm
>   - Version 2.4.114 has been upgaded to version 2.4.118.

Reviewer: ```suggestion
- kmod
  - Version 30 has been upgraded to version 31.
- f2fs-tools
  - Version 1.14.0 has been upgraded to version 1.16.0.
- libdrm
  - Version 2.4.114 has been upgraded to version 2.4.118.
```

- device-mapper
  - Version 2.02.98 has been upgraded to version 2.03.22.
- KDBus
  - KDBus has been supported on the upgraded glib 2.78.4.

#### Fixes
- sessiond
 - The dependency with libsessiond has been removed to reduce the memory usage when it is not used.
- storaged
 - Tizen.System.Feedback issue that does not detect 'unmountable event' has been fixed.
- sensord

>  - The exception handling code when listener connection is failed has beed added.

Reviewer: ```suggestion
 - The exception handling code when listener connection is failed has been added.
```

  - New APIs for video codec type settings have been added.
- Media Content
  - Pinyin related defines have been removed.
  - Some APIs with low usability have been deprecated.
  - Bookmark, playlist, tag APIs have been deprecated.
- Metadata Editor
  - All APIs have been deprecated.
- Image Util
   - Remove deprecated APIs from Tizen 5.0 and 5.5.
- MediaController
  - Remove deprecated APIs from Tizen 5.0 and 5.5.
- MediaVision
  - Deprecated legacy face, image, surveillance APIs.
  - Deprecated inference APIs.
- Media Streamer
  - Media Streamer APIs has been removed.

### Network and connectivity

#### New and changed features

- Network
  - New property to provide frequency information in Wi-Fi config APIs has been added.
  - New error type to handle no carrier exception has been added.
  - Supports automatic connection when AP changes from WPA2 to WPA3.
- Bluetooth
  - The API to get the maximum buffer size of L2CAP socket has been added.
- Open source
  - Curl has been upgraded to version 8.5.0.

#### Fixes

- Network
  - The logic to parse device information in multiple device environments has been fixed.
  - D-Bus signal handling code for Wi-Fi has been fixed.

>   - The issues in the code for storing and loading wifi configuration have been fixed.

Reviewer: ```suggestion
  - The issues in the code for storing and loading Wi-Fi configuration have been fixed.
```

  - The issue where unnecessary background scan attempts has been fixed.
- Bluetooth
  - The crash issue in LE scanning has been fixed.
  - The LE scan duty value for low energy has been modified.
    New interval value for low energy is 1400 ms.
    New window value for low energy is 140 ms.
  - Some GATT operation fail issues have been fixed.
  - The PBAP operation fail issue has been fixed.
  - The issue where the previous trigger_bond_info was deleted when requesting bond has been fixed.
  - The issue where event did not occur when CoC connection failed has been fixed.
  - The issue where CoC pending info was not cleared forever has been fixed.
  - The issue where more than 255 GATT attributes could not be supported has been fixed.
  - The issue where server MTU changed callback was not called has been fixed.

### Security

#### New and changed features

- Openssl
  - Openssl has been upgraded to 3.0.13.

### Service framework

#### Fixes

- Push Client
  - Potential defects have been fixed.

### Web framework

#### New and changed features

- WebGPU has been supported.
- Sharescreen API has been supported.

> - Picture in Picture(Document, Media Session API) has been supported.

Reviewer: ```suggestion
- Picture in Picture (Document, Media Session API) has been supported.
```
